docs/collection_metadata.rst

```diff
@@ -15,12 +15,32 @@ Requirements from a collection can be recognized in these ways:
 If any of these files are in the ``build_ignore`` of the collection, it
 will not work correctly.
 
+How-to verify collection-level metadata
+=======================================
+
 Collection maintainers can verify that ``ansible-builder`` recognizes
-the requirements they expect by using the ``introspect`` command. Example:
+the requirements they expect by using the ``introspect`` command.
+
+In order to do that, the collection has to be installed locally.
+
+When installing collections using ansible-galaxy
```

Comment:
We should point to the ansible-galaxy documentation for installing collections instead of maintaining this here.

Reply:
Not sure, it's only a couple of very basic commands here.
The goal is to show users how to try out the command quickly.
I think if we put **only** the ansible-galaxy link, we'll force our users to jump here and there and maybe read what they don't need to read which increases chances of just dropping the whole undertaking.
I've added the link to the guidelines if users need details, it's a good idea, thanks

Reply:
Since this section is directed at collection mantainers, they already know how to install collections so don't need the repeated information. Assuming I'm correct on the primary audience, then I agree with @Akasurde . The only thing that is important to mention here as maybe a note if it's something that tripped people up, is to state if there is a specific directory that the collection needs to be installed in that isn't the 'typical' directory so to speak.

Reply:
persuaded, removed